PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/configuration/AzureConfigurations.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.intellij.configuration;
+
+import com.intellij.openapi.components.PersistentStateComponent;
```

Comment:
how about move this file to com.microsoft.azure.toolkit.intellij.common.settings package (azure-intellij-plugin-lib module) and rename it to ApplicationAzureSettings?
I added a ProjectAzureSettings under it and it aims to manage azure settings on project level.